docs/kyma-environment-broker/03-15-trial-cleanup-cronjob.md

```diff
@@ -0,0 +1,46 @@
+# Trial Cleanup Job
+
+Trial Cleanup Job is a Job that makes the SKR instances with the `trial` plan expire 14 days after their creation.
+Expiration means that the SKR instance is suspended and the `expired` flag is set.
+
+## Details
+
+For each instance meeting the criteria, a PATCH request is sent to Kyma Environment Broker (KEB). This instance is marked as `expired`, and if it is in the `succeeded` state, the suspension process is started. 
+If the instance is already in the `suspended` state, this instance is just marked as `expired`. 
+
+
+### Dry-run mode
+If you need to test the Job, you can run it in the `dry-run` mode.
```

Comment:
```suggestion
### Dry-run mode

If you need to test the Job, you can run it in the `dry-run` mode.
```
That's where the new line was supposed to go in the first place! 
Please double check :D